src/ring_theory/local_properties.lean

```diff
@@ -12,6 +12,7 @@ import ring_theory.localization.integer
 import ring_theory.localization.submodule
 import ring_theory.nilpotent
 import ring_theory.ring_hom_properties
+import algebra.module.localized_module
```

Comment:
Somewhat pedantic: could you move this line so the imports are sorted alphabetically?